src/azul/openapi/schema.py

```diff
@@ -138,7 +139,7 @@ def array(item: TYPE, *items: TYPE, **kwargs):
     return array_type(make_type(item), *map(make_type, items), **kwargs)
 
 
-def enum(*items: PrimitiveJSON, type_: TYPE = None) -> JSON:
+def enum(*items_: PrimitiveJSON, items: List[str] = None, type_: TYPE = None) -> JSON:
```

Comment:
```suggestion
def enum(*items_: PrimitiveJSON, /, *, items: List[str] = None, type_: TYPE = None) -> JSON:
```

Reply:
```suggestion
def enum(*items_: PrimitiveJSON, items: Sequence[str] = None, type_: TYPE = None) -> JSON:
```

Reply:
The first suggestion wont work ask written, as multiple * parameters are not permitted.

Reply:
also `/` must proceed a `*`, we might want to mark this to be inspected with the schema helpers issue. All usages of `enum()` use positional args.

Reply:
That makes sense. The intent behind the first suggestion was to prevent call sites with both positional arguments AND the `items=` keyword argument.

Reply:
But we can do that in code.